docs/reference/rest-api/security/create-roles.asciidoc

```diff
@@ -50,6 +50,9 @@ privilege or action.
 `cluster`:: (list) A list of cluster privileges. These privileges define the
 cluster level actions that users with this role are able to execute.
 
+`description`:: (string) Allows providing a brief description of the role.
+The maximum length is `1000` chars.
```

Comment:
I can't find anywhere else in our docs where we state the max length of a field - is there a reason why you want to do this here?

Reply:
I think it makes sense to callout max limitation in our REST API docs. But, looking at other places, it is for sure not consistent. So I'm open to suggestion here. 
FWIW We state the max length of the role name [here](https://www.elastic.co/guide/en/elasticsearch/reference/8.14/defining-roles.html#:~:text=Role%20names%20must%20be%20at,trailing%20whitespace%20is%20not%20allowed.), but I'm not sure if that's good place  for mentioning `description` field and its limitations.

Reply:
I'm not opposed to more info - we can leave it in for now 👍